README.md

```diff
@@ -127,7 +99,70 @@ from R2-scores to Mean Absolute Scaled Error.
 **Regressive Models:** Possibility to predict a time series from several other time series 
 (e.g., external regressors), using arbitrary regressive models.
 
+**Multivariate Support:** Tools to create, manipulate and forecast multivariate time series.
+
 ## Contribute
 
 The development is ongoing, and there are many new features that we want to add. 
 We welcome pull requests and issues on github.
+
+Before working on a contribution (a new feature or a fix) make sure you can't find anything related in [issues](https://github.com/unit8co/darts/issues). If there is no on-going effort on what you plan to do then we recommend to do the following:
```

Comment:
Could we add a word that people try to discuss issues with us before spending time on them right away? Perhaps just chaing point 1:
```
1. Create an issue, and if possible wait for it to be discussed.
```

Reply:
sure was thinking about it as well